src/api/spec/controllers/group_controller_spec.rb

```diff
@@ -105,7 +105,21 @@
     end
 
     context 'when the group does not exist' do
-      before { put :update, body: '<group><title>new_group</title></group>', params: { title: 'new_group', format: :xml } }
+      let(:new_member) { create(:confirmed_user) }
```

Comment:
```suggestion
```
This `let` is redundant. See line 77. We can get rid of it.

Reply:
Alright